users_ldap_groups/models/res_company_ldap_operator.py

```diff
@@ -20,17 +20,17 @@ def operators(self):
         """Return names of function to call on this model as operator"""
         return ("contains", "equals", "query")
```

Comment:
why not: 

```suggestion
        return ("_contains", "_equals", "_query")
```
to be constitent with the docstring ?

or adapt this docstring (l20)

Reply:
Those are the operators and the `_` simply makes the function private. This function is called in `class ResCompanyLdapGroupMapping` for `fields.Selection()`. Changing the format of the operator naming would affect on how the value is saved in the database and, therefore, brake every existing setting on instances updating the module. The `_` prefix simply makes the function private, but the operator is a bit different concept despite having a corresponding operator, isn't it? You could suggest a better docsting, though.

Reply:
Thanks for the explaination 
I just suggested a docstring change

Reply:
Thanks! That seems ok even without further explanations on the docstring.